### PR TITLE
WIP Issue#127 ConfigureQuartz. Added a basic working configuration. Added…

### PR DESCRIPTION
**Pull request by Dandria1802**

### CODE REVIEW

File: home-application/src/main/resources/config/tables_postgres.sql [hunk 1]
```diff
@@ -0,0 +1,187 @@
+-- Thanks to Patrick Lightbody for submitting this...
```
Comment (kh0ma) [inline home-application/src/main/resources/config/tables_postgres.sql]: Can we get rid of this sql file?

File: home-application/src/main/resources/config/application.properties [hunk 2]
```diff
@@ -1,3 +1,8 @@
 management.server.port=${MANAGEMENT_SERVER_PORT:8090}
 management.endpoints.web.exposure.include=health
 management.endpoint.health.show-details=always
+
+spring.quartz.job-store-type = jdbc
+spring.quartz.properties.org.quartz.jobStore.class=org.quartz.impl.jdbcjobstore.JobStoreCMT
+spring.quartz.properties.org.quartz.jobStore.driverDelegateClass=org.quartz.impl.jdbcjobstore.PostgreSQLDelegate
+spring.quartz.jdbc.initialize-schema=always
```
Comment (kh0ma) [inline home-application/src/main/resources/config/application.properties]: What other options do we have?

File: home-application/src/main/java/com/softserveinc/ita/homeproject/application/quartz/triggers/DataRetrievalTrigger.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.scheduling.quartz.SimpleTriggerFactoryBean;
+
+@Configuration
+public class DataRetrievalTrigger {
+    @Bean(name = "dataRetrievalTriggers")
+    public SimpleTriggerFactoryBean triggerMemberStats(@Qualifier("dataRetrieval") JobDetail jobDetail) {
+        return QuartzConfig.createTrigger(jobDetail, 60000, "Data retrieval job trigger");
```
Comment (kh0ma) [inline home-application/src/main/java/com/softserveinc/ita/homeproject/application/quartz/triggers/DataRetrievalTrigger.java:14]: Is there any example of a particular job? It isn't clear from naming. I would like to see how for instance notifications job will be set.

**Review by kh0ma — CHANGES_REQUESTED** [review]

File: home-application/src/main/resources/config/tables_postgres.sql (hunk 1 above)
Comment (Dandria1802) [reply home-application/src/main/resources/config/tables_postgres.sql]: Yes.

File: home-application/src/main/java/com/softserveinc/ita/homeproject/application/quartz/triggers/DataRetrievalTrigger.java (hunk 3 above)
Comment (Dandria1802) [reply home-application/src/main/java/com/softserveinc/ita/homeproject/application/quartz/triggers/DataRetrievalTrigger.java:14]: I will change the class names. These are really incomprehensible names.

Comment (Dandria1802) [reply home-application/src/main/java/com/softserveinc/ita/homeproject/application/quartz/triggers/DataRetrievalTrigger.java:14]: Yes, I'll make a working example.

**Pull request merged**